docs/read_the_docs/source/advanced_tutorial/index.rst

```diff
@@ -12,9 +12,9 @@ If your changes are intended to be merged into the ``develop`` branch of the `Re
 .. toctree::
    :maxdepth: 2
 
-   installer_setup
```

Comment:
Removing installer setup. Does anyone actually need/use this? Seems like we only need the OS CLI...